src/OpenTelemetry.Instrumentation.AspNet/CHANGELOG.md

```diff
@@ -5,6 +5,8 @@
 * AspNetInstrumentation sets ActivitySource to activities created outside
   ActivitySource.
   ([#1515](https://github.com/open-telemetry/opentelemetry-dotnet/pull/1515/))
+* Added SpanHelper class as a link to the the project.
```

Comment:
Should this be in CHANGELOG?

Reply:
we only need in APIs changelog, not in instrumentations. will do a cleanup